src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,14 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.serialized.size")
```

Comment:
Should this also include `shredding`, so `json.shredding.serialization.size` ?
And then method like `jsonShreddingSerializedSize()`